### PR TITLE
Bugfix/cldsrv 275/bump utapi

### PR DESCRIPTION
**Pull request by tmacro**
# Pull request template

### Motivation and context

Why is this change required? What problem does it solve?

### Related issues

Please use the following link syntaxes #600 to reference issues in the
current repository

### Add tests to cover the changes

New tests added or existing tests modified to cover all changes

### Sign your work

In order to contribute to the project, you must sign your work
https://github.com/scality/Guidelines/blob/master/CONTRIBUTING.md#sign-your-work

Thank you again for contributing! We will try to test and integrate the change
as soon as we can.

### CODE REVIEW

**Review by naren-rajendran — APPROVED** [review]

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -15,21 +16,21 @@ jobs:
...
-      - name: Create Release
-        uses: softprops/action-gh-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          name: Release ${{ github.event.inputs.tag }}
-          tag_name: ${{ github.event.inputs.tag }}
-          generate_release_notes: true
-          target_commitish: ${{ github.sha }}
+#  github-release:
```
Comment (jonathan-gramain) [inline .github/workflows/release.yaml]: Can we remove this instead of commenting out if it's not needed?

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -2,6 +2,7 @@
 name: release
 
 on:
+  push: {}
```
Comment (jonathan-gramain) [inline .github/workflows/release.yaml]: Was this meant to be temporary only?

File: .github/workflows/release.yaml (hunk 1 above)
Comment (tmacro) [reply .github/workflows/release.yaml]: The third commit will be dropped before merge, I needed a s3c image built to test the integration tests

File: .github/workflows/release.yaml (hunk 2 above)
Comment (tmacro) [reply .github/workflows/release.yaml]: see above

**Review by jonathan-gramain — APPROVED** [review]

**Pull request merged**